src/main/java/com/ebivariation/contigalias/dus/NCBIBrowser.java

```diff
@@ -0,0 +1,69 @@
+package com.ebivariation.contigalias.dus;
+
+import org.apache.commons.net.ftp.FTPFile;
+
+import java.io.IOException;
+
+public class NCBIBrowser extends FTPBrowser {
+
+    public static final String NCBI_FTP_SERVER = "ftp.ncbi.nlm.nih.gov";
+    public static final String PATH_GENOMES_ALL = "/genomes/all/";
+
+    public void connect() throws IOException {
+        super.connect(NCBI_FTP_SERVER);
```

Comment:
You don't need to use `super` here unless you have a method with the same signature in the super-class and sub-class
```suggestion
        connect(NCBI_FTP_SERVER);
```

Reply:
I put it there as it makes it easier to know what's happening